osidb/tests/factories.py

```diff
@@ -350,6 +351,35 @@ class Meta:
         "creation_time": "2006-03-30T11:56:45Z",
     }
 
+    @staticmethod
+    def prepare_order_text(kwa):
+        """
+        Mock "order" and "text" fields in a way that passes model validation.
+        """
+        if "order" not in kwa:
+            if kwa["flaw"].comments.count() == 0:
+                # Comment 0 would be the flaw description.
+                # Creating a proper comment here, so starting with 1.
+                order = 1
+            else:
+                # Description is optional, so just using .count() doesn't work consistently.
+                order = 1 + kwa["flaw"].comments.aggregate(Max("order"))["order__max"]
+            kwa["order"] = order
```

Comment:
Can't this just be https://factoryboy.readthedocs.io/en/stable/reference.html#factory.Sequence ?

Reply:
It can! Since we are removing the "order" validation, it doesn't matter what's in there. I can remove the 3 new methods and make the factory changes really simple then.

Reply:
Resolved in commit a3415f9b5cde43c7e4fb3ac8476e8791fd710365